src/org/usfirst/frc/team2335/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,83 @@
+package org.usfirst.frc.team2335.robot.subsystems;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.TalonSRX;
+import edu.wpi.first.wpilibj.Victor;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+
+public class Shooter extends Subsystem 
+{
+	//Motors for the shooter
+	private Victor feederMotor;
+	private TalonSRX shooterMotor;
+	private TalonSRX intakeMotor;
+	
+	public double motorSpeed = -0.68;
+	
+	public Shooter()
+	{
+		super();
+		
+		//Defines motors for the shooter
+		feederMotor = new Victor(Robot.FEEDER_MOTOR);
+		shooterMotor = new TalonSRX(Robot.SHOOTER_MOTOR);	
+		intakeMotor = new TalonSRX(Robot.INTAKE_MOTOR);
+	}
+		
+	//Shooter motor methods
+	public void shootBall()
+	{
+		//TODO: Find best speed for shooter motor
+		shooterMotor.set(motorSpeed);
+	}
+	
+	public void speedUp()
+	{
+		motorSpeed -= 0.05;
```

Comment:
OK, thanks. Looking at this again (and with your explanation) this is clear, but some function documentation could help make things extra clear for future years' team members.